routes/target.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var router = express.Router();
```

Comment:
It’s good practice to use `const` for variables that are not reassigned (like express and router). This avoids accidental reassignments.

Reply:
Noted and updated.